packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -8,25 +8,25 @@ import 'package:{{pluginProjectName}}/{{pluginProjectName}}.dart';
 {{/withPluginHook}}
 
 {{^withDriverTest}}
-void main() => runApp(new MyApp());
+void main() => runApp(MyApp());
 {{/withDriverTest}}
 {{#withDriverTest}}
 void main() {
   // Enable integration testing with the Flutter Driver extension.
   // See https://flutter.io/testing/ for more info.
   enableFlutterDriverExtension();
-  runApp(new MyApp());
+  runApp(MyApp());
 }
 {{/withDriverTest}}
 
 {{^withPluginHook}}
 class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
-  Widget build(BuildContext context) {
-    return new MaterialApp(
+  Widget build(BuildContext context) =>
```

Comment:
we'd rather stick with the "return" here because it's more familiar to people who haven't used Dart before.

Reply:
I'm happy to change this but the arrow syntax was already being used in other places already
 - https://github.com/flutter/flutter/pull/15502/files#diff-6ba688473d3e61aca6d653e5b6528200L11
 - https://github.com/flutter/flutter/pull/15502/files#diff-6ba688473d3e61aca6d653e5b6528200L60
 - https://github.com/flutter/flutter/pull/15502/files#diff-6ba688473d3e61aca6d653e5b6528200L132

So I thought it'd make more sense to unify all usages so that at least it's consistent, what are your thoughts @Hixie?

Reply:
For code in the flutter repo it is the convention, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods

Reply:
Thanks for the link!

Reply:
Fixed in 96f083733da0d5e913db316354f72e98504ff674